nmeaoutput.py

```diff
@@ -71,7 +71,7 @@ def gen_gga(time_t, lat, lng, fix_quality, num_sats, hdop, alt_m, geoidal_sep_m,
     result = 'GPGGA,%s,%s,%s,%s,%s,%d,%02d,%.1f,%.1f,M,%.1f,M,%s' % (hhmmssss, lat_format, lat_pole_prime, lng_format, lng_pole_prime, fix_quality, num_sats, hdop, alt_m, geoidal_sep_m, dgps_format)
     crc = checksum(result)
 
-    return '$%s*%0.2X' % (result, crc)
+    return '$%s*%0.2X\r\n' % (result, crc)
```

Comment:
Thanks for the fix. This will give a double newline on serial, but adding the newline in the gengga should be fine. Remove the newline here, and I will merge it:
https://github.com/waterlinked/examples/blob/2f3602671c79f945d4cf47990ef47f8e251c3dcc/nmeaoutput.py#L157

Reply:
Done.